### ISSUE TITLE
[Header] Header displays over body

### ISSUE THREAD

**Issue by MaryWeeb** [material-components/material-components-web#6718]
## Bug report
The app-top-bar displays over the main body, rather than before the main body

### Steps to reproduce
1. Include the example HTML
2. View the page

### Actual behavior
My page body is partially behind the header

### Expected behavior
The header is placed above the page body

### Your Environment:

| Software         | Version(s) |
| ---------------- | ---------- |
| MDC Web          | latest
| Browser          | chromium latest
| Operating System | arch linux latest

**Comment by asyncLiz** [comment]
Do you have the `mdc-top-app-bar--fixed-adjust` class added to your main content element? The documentation is not very clear that this is required for both the regular and fixed variants, our examples could be updated to show this.

Here is an [example with the regular app bar](https://jsbin.com/pomazuzaxi/edit?html,output).

```html
<header class="mdc-top-app-bar">
  ...
</header>
<main class="mdc-top-app-bar--fixed-adjust">
  ...
</main>
```

Don't forget to substitute that class for one of the other "fixed-adjust" classes depending on whether or not the app bar is short, dense, or prominent as well.

**Comment by MaryWeeb** [comment]
That seems to work, though the app bar still goes offscreen on the right because of its `position:fixed` and `width:100%` despite starting a bit from the left.

![Screenshot_20201229_091806](https://user-images.githubusercontent.com/16361449/103269780-cc931980-49b6-11eb-8dfa-dd9936300944.png)

**Comment by asyncLiz** [comment]
That looks like it might be the default browser's user agent stylesheet. All browsers add a default margin to the body. You can remove this with the following global style:

```css
body {
  margin: 0;
}
```

There are other alternative ways around this, such as setting your application's container to the full viewport width and height (`100vw`, `100vh`). Since MDC does not know how your application is being mounted into the body or what framework might be in use, these styles are left to the user to control.

**Comment by asyncLiz** [comment]
Fixed in #6726
